resources/views/contacts.blade.php

```diff
@@ -83,13 +83,15 @@
                         <div class="staff__block_item">
                             <div class="staff__item_top">
                                 <h3 class="staff__item_bold text-20">{{ $item->{'position_'.app()->getLocale()} }}</h3>
-                                <h4 class="staff__item_bold text-18">
-                                    {!! $item->{'name_'.app()->getLocale()} ?? ' ' !!}
-                                </h4>
+                               <h4 class="staff__item_bold text-18 {{ empty($item->{'name_'.app()->getLocale()}) ? 'hidden' : '' }}">
+                                   {!! $item->{'name_'.app()->getLocale()} ?? 'No name available' !!}
```

Comment:
@hambardzumianvahag Please use language key instead of hard coded text.
Instead of **'No name available'** use for example **__('no_name_available')**. And in each directory of **resources/lang** add a value with that key.
For example in **resources/lang/en** folder, in **contact.php** add a value like this
`'no_name_available' =>  'No name available'`

Reply:
I agree with you but in this case I give span class "hidden" which will hide span(display:'none') and this is wish of our client to hide text when its empty.